### PR TITLE
HOSS2-337 - Encrypt PII in Mongo

### PR DESCRIPTION
**Pull request by burns238**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -159,6 +159,9 @@ mongodb {
   # ttl = 1 hour
   ttl.seconds = 3600
   collectionName = form-query
+  encryption {
+    key = "VqmXp7yigDFxbCUdDdNZVIvbW6RgPNJsliv6swQNCL8="
```
Comment (liampan) [inline conf/application.conf:163]: so is this encrypted?

File: test/services/SessionCacheServiceSpec.scala [hunk 2]
```diff
@@ -27,30 +27,41 @@ import repositories.SessionCacheRepository
...
     reset(mockRepo)
     super.beforeEach
   }
 
+  private val secretKey = "VqmXp7yigDFxbCUdDdNZVIvbW6RgPNJsliv6swQNCL8="
   val formModel = StatusCheckByNinoFormModel(
     nino = NinoGenerator.generateNino,
-    givenName = "Pan",
-    familyName = "Oldfield",
+    givenName = "Jimmy",
```
Comment (liampan) [inline test/services/SessionCacheServiceSpec.scala:60]: ty

**Review by liampan — APPROVED** [review]

File: conf/application.conf (hunk 1 above)
Comment (burns238) [reply conf/application.conf:163]: This isn't, but when we have an actual key to configure in our app config in production, that will be encrypted by the infrastructure team so we'll need to set a ticket up when we change the app config

Comment (liampan) [reply conf/application.conf:163]: gotcha

**Pull request closed without merging**